portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,19 +33,24 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> messages = new ArrayList<String>();
-  
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
     //Get input from form.
-    String text = getParameter(request, "comments", "");
+    String text = request.getParameter("comments");
+
+    //-----------------------------
+    //Populate list with data.
+    //messages.add(text);
+    //-----------------------------
```

Comment:
Save to delete this right?

Reply:
Yes! It was commented out to be deleted so it can be removed.